README.md

```diff
@@ -9,12 +9,12 @@ renovate.json
 ```
 {
   "extends": [
-    "github>CondeNast/renovate-config:unpublishSafe",
     "github>CondeNast/renovate-config:forceCondenastUpdates",
     "github>CondeNast/renovate-config:automergeMinor",
     "github>CondeNast/renovate-config:condenastStabilityDays",
-    "github>CondeNast/renovate-config:dontAutomergeNode",
-    "github>CondeNast/renovate-config:dontAutomergeTestFrameworks",
+    "github>CondeNast/renovate-config:dontAutomergeMajor(help wanted)",
+    "github>CondeNast/renovate-config:dontAutomergeNode(help wanted)",
+    "github>CondeNast/renovate-config:dontAutomergeTestFrameworks(help wanted)",
```

Comment:
with Rocket we will want to use `:heart: help wanted` rather than just `help wanted`